docker-compose-local.yml

```diff
@@ -16,50 +16,6 @@ services:
       - "${TMPDIR:-/tmp/localstack}:/var/lib/localstack"
       - "/var/run/docker.sock:/var/run/docker.sock"
 
-  hmpps-auth:
```

Comment:
Why did you remove this?

Reply:
These are moved in the `docker-compose.yml`, this is the local one that we use to run the API against `dev` environment but running DB/localstack in docker-compose. The `docker-compose.yml` has everything.